addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -116,5 +116,58 @@ const PopupWidget = publicWidget.Widget.extend({
 
 publicWidget.registry.popup = PopupWidget;
 
+// Extending the popup widget with cookiebar functionality.
+// This allows for refusing optional cookies for now and can be
+// extended to picking which cookies categories are accepted.
+publicWidget.registry.cookies_bar = PopupWidget.extend({
+    selector: '#website_cookies_bar',
+    cookieDurationDays: 365,
```

Comment:
Why not the same as the cookies bar 999 (see its data attribute `data-consents-duration="999"`), 

So no need of this property just inline `this.$el.find('.modal').data('consentsDuration');`

Because right now, if one is accepting the cookies bar, he has 2 cookies, the first about showing or not the cookies bar, set for 999 days, and this one set for 365 days, it means that after a year, his choice about cookies preference will not be saved anymore, but he still won't see the cookies bar.

Another reason to use the same cookie imo (see below remark)